blog.DAL/Entities/Image.cs

```diff
@@ -8,6 +9,7 @@ public class Image
         {
             [Key]
             public int ImageId { get; set; }
+            [Required, JsonProperty("Url:")]
```

Comment:
JsonProperty("Url:")-ի ինչ իմաստ ունի?

Reply:
Vor json tesqov urly uxarkeluc haskana vor property - in enq poxancum. Kareliya chgrel ?

Reply:
էս դեպքում json-ը սենց տեսքա ունենլու "Url:": "some url" 
առանց գրելու պիտի որ նորմալ աշխատի